README.md

```diff
@@ -198,15 +199,18 @@ Currently, the following clients are available for use:
 - `ANDROID_MUSIC`
   - ✔ Opus formats.
   - ❌ No playlist/livestream support.
+- `ANDROID_VR`
+  - ✔ Opus formats.
+  - ❌ No 'YouTube Kids' video support.
 - `MEDIA_CONNECT`
   - ❌ No Opus formats (requires transcoding).
   - ❌ No mix/playlist/search support.
 - `IOS`
   - ❌ No Opus formats (requires transcoding).
 - `TVHTML5EMBEDDED`
   - ✔ Opus formats.
-  - ✔ Age-restricted video playback.
   - ❌ No playlist support.
+  - ❌ Cannot be used for playback when logged out.
```

Comment:
```suggestion
  - ❌ Requires sign in.
```

Just for conciseness and to somewhat explicitly link it to the issues we've been seeing.